core/deployment/src/main/resources/dev-ui/qwc-aiservices.js

```diff
@@ -0,0 +1,85 @@
+import { LitElement, html, css} from 'lit';
+import { JsonRpc } from 'jsonrpc';
+import '@vaadin/icon';
+import '@vaadin/button';
+import '@vaadin/text-field';
+import '@vaadin/text-area';
+import '@vaadin/form-layout';
+import '@vaadin/progress-bar';
+import '@vaadin/checkbox';
+import '@vaadin/grid';
+import 'qui-alert';
+import { columnBodyRenderer } from '@vaadin/grid/lit.js';
+import '@vaadin/grid/vaadin-grid-sort-column.js';
+
+import {aiservices} from 'build-time-data';
+
+
+export class QwcAiservices extends LitElement {
+
+    static styles = css`
+        .button {
+            cursor: pointer;
+        }
+        .clearIcon {
+            color: orange;
+        }
```

Comment:
I don't think this is used. Also, if using colors (like orange) rather refer to the theme vars (--lumo-warning-text-color)